framework/src/media/MediaRecorderImpl.cpp

```diff
@@ -554,15 +557,15 @@ void MediaRecorderImpl::capture()
 			if (written == EOF) {
 				meddbg("MediaRecorderImpl::capture() failed : errno : %d written : %d\n", errno, written);
 				RecorderWorker& mrw = RecorderWorker::getWorker();
-				mrw.enQueue(&MediaRecorderImpl::stopRecorder, shared_from_this(), -1);
+				mrw.enQueue(&MediaRecorderImpl::stopRecorder, shared_from_this(), false);
```

Comment:
Why do you change this value?
stopRecorder has int argument, but you transfer bool parameter.

Reply:
Due to merging... Reverted.